src/main/kotlin/no/fdk/fdk_reasoning_service/rdf/CV.kt

```diff
@@ -11,10 +11,6 @@ class CV {
 
         val hasCompetentAuthority: Property = ResourceFactory.createProperty("${uri}hasCompetentAuthority")
         val ownedBy: Property = ResourceFactory.createProperty("${uri}ownedBy")
-        val playsRole: Property = ResourceFactory.createProperty("${uri}playsRole")
```

Comment:
Disse utgår?

Reply:
Ja, siden vi nå går over til å kun kalkulere enkeltressurser så trengher ikke reasoning-service å trekke ut enkeltressurser fra hver katalog, bruker alltid bare input-modellen as-is.

De fleste av disse ble brukt til akkurat det, det som gjenstår er orgPath og themePath, som legges til av reasoning-service, og predikater som brukes for å gjenkjenne organisasjoner.